data/language/english_uk.txt

```diff
@@ -4015,7 +4016,7 @@ STR_5709    :Rename Group
 STR_5710    :Group name
 STR_5711    :Enter new name for this group:
 STR_5712    :Can't modify permission that you do not have yourself
-STR_5713    :Kick Player
+STR_5713    :Can't remove default group
```

Comment:
For my second commit, I picked this string id so that my string could be grouped together with the other strings related to modifying player groups. I added my string id `STR_CANT_REMOVE_DEFAULT_GROUP = 5713,` before I went to the language file and saw that there was already a `STR_5713`. Since string id `5713` was missing before I added mine, that meant it was unused. And there was no reason to reserve it, so I decided to go ahead with it.

ef4fc84 shows that `STR_ACTION_KICK_PLAYER` was moved from `5713` to `5659`.